src/tools/ip/quicip.h

```diff
@@ -0,0 +1,174 @@
+/*++
+
+    Copyright (c) Microsoft Corporation.
+    Licensed under the MIT License.
+
+Abstract:
+
+    A client helper for resolving the public IP address.
+
+TODO:
+
+    Don't use the QUIC_ADDR format, as it's not cross-platform. Use the same
+    format as the QUIC spec's preferred address TP.
+
+--*/
+
+#include <msquichelper.h>
```

Comment:
By doing this, we'll need to add msquichelper.h to the public headers list
https://github.com/microsoft/msquic/blob/master/src/bin/CMakeLists.txt#L41

Also means msquichelper.h needs to be updated to not include quic_platform.h, msquicp.h, and not use any functions needed by those 2 headers either.

Reply:
Fixed this another way.